python/smashbox/multiprocessing_engine.py

```diff
@@ -113,16 +113,22 @@ def supervisor(steps):
         if _smash_.DEBUG:
             log('start',_smash_.supervisor_step.value,_smash_.steps)
 
+        _smash_.N_STEPS = max(
+            int(config.get('smashbox_steps_limit', -1)), #this can be configured using [-o smashbox_steps_limit=N]
+            _smash_.N_STEPS)
+
```

Comment:
@moscicki What do you think ? Could we define number of steps in config file? Use case is that you want to sync many many files, and have a synchronisation between them for each step.